### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by cmm-fi** [Take434/Appollon#20]
Hey,
this is just a question, you do not need to answer it. 

the root page.tsx:
Does your app ever use the lines 

13 to 18

?
I am just asking myself wrt. to the App-Router concept of next.js if it will be okay to diretcly intergrate the login-components more concrete the userloggedIN-logic into the root-page. 
Or do you have any argument against it?

CMM

**Comment by Take434** [comment]
Hi,
as you have correctly noticed the app doesnt use the root page.tsx for anything at all.
It simply redirects the user to the login page.

This is not necessarily a pattern from the next.js App-Router but something of a side effect.
The reason why it was implemented that way has to do with the layout for logged in users.
When starting to develop the application the original idea was to serve the home page now found under `/home` at the root of the project. This necessitated a different page for the login logic.
While developing we then discovered, that it was a common pattern to have all pages with a different layout, meaning the pages a user can see while being logged in, in a subdirectory which also serves a new layout.
But this meant, that it was no longer possible to have the root `page.tsx` of the project serve as the home page for logged in users. 

To answer your question, while it would totally be possible to refractor the app a little and serve the login logic at the root, we chose to focus on more important features and bug fixes. 

I hope this answers your question,
kind regards
Tim
